packages/theme/css/characteristics/status.css

```diff
@@ -20,4 +20,9 @@
   --salt-status-success-background-selected: var(--salt-palette-success-background-selected);
   --salt-status-warning-background-selected: var(--salt-palette-warning-background-selected);
   --salt-status-error-background-selected: var(--salt-palette-error-background-selected);
+    
+  /* Deprecate all below */
+  --salt-status-success-foreground: var(--salt-palette-success-foreground);
+  --salt-status-warning-foreground: var(--salt-palette-warning-foreground);
+  --salt-status-error-foreground: var(--salt-palette-error-foreground);
```

Comment:
```suggestion
```
These can be removed from here since they are in deprecated.

Reply:
done